xenserver/tests/fake_xen_server.py

```diff
@@ -5,6 +5,9 @@
 import xenapi
 
 
+MEM_OVERHEAD = 1*1024*1024*1024
```

Comment:
1 GiB. There's an "overhead" field in the XenServer API and from looking at our servers that seems to be in the range 1-1.5GiB. All that really matters is that we have `memory_free` lower than `memory_total`.